libcalico-go/config/crd/crd.projectcalico.org_felixconfigurations.yaml

```diff
@@ -610,6 +610,20 @@ spec:
                 description: FlowLogGoldmaneServer is the flow server endpoint to
                   which flow data should be published.
                 type: string
+              flowLogsPolicyEvaluationMode:
+                description: |-
+                  FlowLogsPolicyEvaluationMode defines how policies are evaluated and reflected in flow logs.
+                  OnNewConnection - In this mode, staged policies are only evaluated when new connections are
```

Comment:
You will have to regenerate the YAMLs so that they reflect the options on OSS. `OnNewConnection` will not exist here

Reply:
Updated.